lib/tsUtils.tsx

```diff
@@ -8,3 +8,23 @@ export function hasOwnProperty<X extends Record<string, unknown>, Y extends Prop
 export const isServer = (): boolean => {
   return typeof window === "undefined";
 };
+
+/**
+ * This function checks if a given email is a government valid email.
+ * And it returns true if the email is a valid GC email otherwise false.
+ * @param email A valid government email
+ * @param domains The list of GC domains
+ * @returns {boolean} The validation result
+ */
+export const isValidGovEmail = (email: string, domains: string[]): boolean => {
```

Comment:
Not sure this file is the appropriate place for this function. Perhaps this is better placed in the validation.tsx

Reply:
Yeah i can be moved in the validation.tsx file as well.